_docker/drupal/drupal-filesystem/web/config/sync/core.entity_view_display.node.assembly_page.teaser.yml

```diff
@@ -7,6 +7,16 @@ dependencies:
     - field.field.node.assembly_page.field_meta_tags
     - field.field.node.assembly_page.field_sections
     - field.field.node.assembly_page.field_share_image
+    - field.field.node.assembly_page.field_tax_audience_segment
```

Comment:
@jordanpagewhite Same here. Just double-checking that we're not changing more than we expect too with this PR.

Reply:
Yep, this is also good